### ISSUE TITLE
Cypress could not verify that this server is running

### ISSUE THREAD

**Issue by p-a-s-c-a-l** [clarity-h2020/csis-technical-validation#5]
![grafik](https://user-images.githubusercontent.com/1788226/73525646-eaa37300-440f-11ea-8c4b-7d3b57a0dba4.png)

Setting the baseUrl programmatically instead of specifying it in cypress.json does not lead to the 'Cypress could not verify that this server is running' error. 😒

**Comment by p-a-s-c-a-l** [comment]
CI tests fail because of this problem:

```
Cypress could not verify that this server is running:

  > https://csis.myclimateservice.eu/

We are verifying this server because it has been configured as your `baseUrl`.

Cypress automatically waits until your server is accessible before running tests.

We will try connecting to it 3 more times...
(node:12948) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
(node:12948) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
We will try connecting to it 2 more times...
(node:12948) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
(node:12948) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
We will try connecting to it 1 more time...

(node:12948) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
(node:12948) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 pipe listeners added to [Request]. Use emitter.setMaxListeners() to increase limit
Cypress failed to verify that your server is running.

Please start this server and then run Cypress again.
```

**Comment by p-a-s-c-a-l** [comment]
This problem occurs only with cisis. not with profiles. probably because .cisis redirects to .profiles CAS service: `https://profile.myclimateservices.eu/cas/login?gateway=true&service=https%3A//csis.myclimateservice.eu/casservice%3Fdestination%3D/`'

**Comment by p-a-s-c-a-l** [comment]
workaround implemented in 33bca36

**Comment by rongrong-lu-awx** [comment]
I have the same problem,because I delete the regression url[v2]
![image](https://user-images.githubusercontent.com/78594169/129683386-60be105b-ac3d-46f1-8599-9683fe935f19.png)
